### ISSUE TITLE
Weird Caching issue in incognito mode + Internal Server Error spam

### ISSUE THREAD

**Issue by AmansRevenger** [tidusjar/Ombi#1637]
#### Ombi build Version:

V 3.0.2234

```
fail: Ombi.Api.Api[1000]
     StatusCode: InternalServerError, Reason: Internal Server Error
```

#### Problem Description:

Console Spams above Error all the time when I select ANYTHING from the Headers in the Settings menu
Everything still works as far as I can see, but there must be something wrong in the background.

Another thing is this

![image](https://user-images.githubusercontent.com/18306225/32119478-ad48d53a-bb55-11e7-9edb-30973672ef12.png)

basically it's a serious caching issue where it puts the most recent clicked "frame" (?) on top of the page and everything else gets packed below it.

I had it happen once in normal browsing, but it happens basically all the time in incognito mode

Here is a gif showing what I mean
 
![ombi plz](https://user-images.githubusercontent.com/18306225/32119812-bc44124c-bb56-11e7-89a5-d3691e930843.gif)

**Comment by PotatoQuality** [comment]
pleas uwe the whole issue format instead of parts  from it

**Comment by AmansRevenger** [comment]
Half the Issue format isnt relevant to this issue, so I'm not resubmitting, Just trying to help and this is what I get?

Nice.

**Comment by PotatoQuality** [comment]
for us its relevant to have all of the information instead of parts from it.
feel free to open an new issue with the compleet issue format so we can proper investigate your problem.

(fyi in the past we did this before with a bot)

second, the first line when you open an issue is:
"Please use the Support / bug report template, otherwise we will close the Github issue !!"

> see you are very closing happy.
point is? its pritty clear to use the whole issue format instead of parts from it ;)

**Comment by tidusjar** [comment]
Are there any errors in the console when this happens?

**Comment by AmansRevenger** [comment]
No, everything seems clear on that part.

**Comment by tidusjar** [comment]
Open the console while recreating the issue. There must be something

**Comment by Codehhh** [comment]
The "weird caching issue" is angular not knowing what to do after an error is thrown

**Comment by tidusjar** [comment]
Any luck?
